website/docs/r/integration_pagerduty.html.markdown

```diff
@@ -12,6 +12,38 @@ Provides a Datadog - PagerDuty resource. This can be used to create and manage D
 
 ## Example Usage
 
+*Note:* In the past, service objects under the `services` key were specified inside the `datadog_integration_pagerduty` resource. This was incompatible with multi-configuration-file setups, where users wanted to have individual service objects controlled from different Terraform configuration files. The recommended approach now is specifying service objects as individual resources using [datadog_integration_pagerduty_service_object](/docs/providers/datadog/r/integration_pagerduty_service_object.html) and adding `individual_services = true` to the `datadog_integration_pagerduty` object.
```

Comment:
Could we specify a provider version here instead of `in the past`? (I assume we'll have to bump to 3.0.0 since we're deprecating something?

Reply:
Fixed.